target/linux/realtek/image/Makefile

```diff
@@ -19,6 +19,12 @@ define Device/Default
 	append-metadata | check-size
 endef
 
+# Use this when U-Boot supports LZMA compression
+define Device/default_lzma
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | uImage lzma
+endef
```

Comment:
I don't think we need this additional definition. I would include the two lines in netgear_nge until we have more users.